### DISCUSSION TITLE
New version 4.0.0 BETA now available for testing!

### DISCUSSION THREAD

**Discussion by bluecaret** [bluecaret/carettab#339] [! News and Announcements]
### I'm thrilled to invite you all to test drive the new version, 4.0.0 of CaretTab, available now in **BETA**!
*NOTE: BETA IS NOW CLOSED. V4.0.1 WILL BE AVAILABLE IN THE STORE WITHIN A FEW DAYS*

I've been busy adding some exciting new features and improvements. Now you can personalize your New Tab even more with an enhanced settings panel, the addition of the Todo, Shape, and Text widgets, and a handy toolbar with tools like AI Chat, calculator, and more. Plus, the ability to add multiples of any widget, and a brand-new positioning system for easier widget placement are sure to make CaretTab even more enjoyable. Weather widgets have had a major revamp, and there's even an option to pull beautiful wallpapers from Pexels.com. And trust me, there's so much more to discover! Check out the [changelog on GitHub](https://github.com/bluecaret/carettab/wiki/Changelog).

Remember, this is a beta version, so please let me know if you come across any bugs or issues or any small suggestions.

I plan on running this beta for **about 1-2 weeks**, depending how it goes, before releasing to production so I hope you all have a chance to give it a good shot! Afterwards the beta will be unpublished again and I'll ask you all to move to the production version.

Thanks again for your help and support, and I'm looking forward to hearing your thoughts on the latest version!

Cheers,
John Hancock (a.k.a. BlueCaret)

**Comment by ryanbuening** [comment]
Nice!

1. Will customizations made to the beta app carry over to the production release?
2. Would you like bugs logged in the comments here or somewhere else?

**Reply by bluecaret** [reply]
1) Good question! Customizations cannot be automatically carried over, but you should be able to use the export/import functions found in settings > extension settings > advanced > export/import

If you've helped beta test and run into issues with this, please feel free to send your export to me at dev@bluecaret.com and I can ensure it gets formatted correctly to be able to be imported to production. This especially may be needed if some bug fixes require tweaks to the  settings keys.

2) You can make a new discussion topic Help and Support if you like, or posting anything in this thread is fine as well. https://github.com/bluecaret/carettab/discussions/categories/help-and-support

Thanks!

**Comment by bluecaret** [comment]
The BETA is now closed as I'm pushing to production now 🙌

Version 4.0.1 will be available in Edge first, probably in the next day or two. It will release in Chrome a few days later.

Can't wait for everyone to try it out!
